src/index.js

```diff
@@ -489,7 +489,7 @@ class SVGIcons2SVGFontStream extends Transform {
         glyphPath.getBounds();
       if (this._options.centerHorizontally) {
         glyphPath.translate(
-          (glyph.width - (bounds.maxX - bounds.minX)) / 2 - bounds.minX
```

Comment:
I'm not 100% on this

To me this solved my issues with misaligned glyphs but it's best if someone that understands this better gives some feedback